R/read-gs.R

```diff
@@ -15,11 +16,200 @@ read_gs <- function(path, verbose = FALSE){
   gs <- read_csv(path, trim_ws = FALSE) |>
     #check format
     check_data_format()
+  #add source attribute
+  attr(gs, "source") <- "Gradescope"
+  
+  gs
+}
+
+
+#'Read in 1 or more files from Grading Platform
+#'
+#'This function will read in input data and combine in the appropriate manner.
```

Comment:
Could this be made more a bit more precise? What is "input data"? What does "combine in an appropriate manner" mean? For the former, how about just "assignment grade data" and we can amend later if we take advantage of the `other_file_paths` arg? I think I'd strike the combine thing since, at least right now, there is no combining happening, is there?